src/app/features/public-form/public-form.component.html

```diff
@@ -58,6 +58,17 @@ <h1 i18n>No access to public form "{{ formConfig?.title }}"</h1>
       </mat-card-content>
 
       <mat-card-footer>
+        @if (validationError) {
+          <div
+            class="flex-row align-center justify-content-center margin-regular color-error"
+            role="alert"
+          >
+            <fa-icon icon="triangle-exclamation" class="color-error"></fa-icon>
+            <span i18n>
+              Some fields are invalid, please check the form and submit again.
+            </span>
```

Comment:
```suggestion
            i18n
          >
            Some fields are invalid, please check the form and submit again.
```

missing a bit of gap between icon and text and in other errors we don't display an icon either, so for simplicity I am removing it here.